### DISCUSSION TITLE
Add command line option for showing unread messages

### DISCUSSION THREAD

**Discussion by EbrietanPhasmid** [d99kris/nchat#308] [Ideas]
It would be really neat if one could call the binary with some option like
`nchat --status` 
to receive a printout of all unread messages including the name of the sender.

It would look something like:

```
Whatsapp
Bob   - 13:23 - Hi!
Alice - 6:02  - This is an example WhatsApp message!

Telegram
John  - 9:10  - And this is a telegram message.
```

**Comment by d99kris** [comment]
Non-interactive features are unfortunately out of scope of the project, so I will proceed to close this discussion. 

Feel free to re-open if you have any follow-up questions though.
